### PR TITLE
Add DNS server stats collector

### PR DESCRIPTION
**Pull request by imposibrus**
Add DNS cache saturation metrics:
```
# HELP mktxp_dns_info DNS info
# TYPE mktxp_dns_info gauge
mktxp_dns_info{property="cache_size",routerboard_address="192.168.88.1",routerboard_name="Sample-Router"} 1.2582912e+07
mktxp_dns_info{property="cache_used",routerboard_address="192.168.88.1",routerboard_name="Sample-Router"} 9.517056e+06

# ....
mktxp_collection_time_total{name="DnsCollector",routerboard_address="192.168.88.1",routerboard_name="Sample-Router"} 103.95883402088657
```

### CODE REVIEW

**Comment by akpw** [comment]
@imposibrus thanks for the PR. Looks good, just a comment, can you pls change the new collector to be disabled by default? Kind of a general policy, also on a cosmetic side probably better renaming DnsCollector to DNSCollector to follow the name pattern of other similar ones.

**Comment by imposibrus** [comment]
@akpw done)

File: mktxp/cli/config/config.py [hunk 1]
```diff
@@ -166,7 +168,7 @@ class MKTXPConfigKeys:
 
     # Feature keys enabled by default
     BOOLEAN_KEYS_YES = {PLAINTEXT_LOGIN_KEY, FE_DHCP_KEY, FE_PACKAGE_KEY, FE_DHCP_LEASE_KEY, FE_IP_CONNECTIONS_KEY, FE_INTERFACE_KEY, 
-                        FE_ROUTE_KEY, FE_DHCP_POOL_KEY, FE_FIREWALL_KEY, FE_NEIGHBOR_KEY, FE_MONITOR_KEY, MKTXP_USE_COMMENTS_OVER_NAMES,
+                        FE_ROUTE_KEY, FE_DHCP_POOL_KEY, FE_FIREWALL_KEY, FE_NEIGHBOR_KEY, FE_DNS_KEY, FE_MONITOR_KEY, MKTXP_USE_COMMENTS_OVER_NAMES,
```
Comment (akpw) [inline mktxp/cli/config/config.py:171]: this should match the config setting, otherwise would cause a mismatch for existing installations

**Comment by akpw** [comment]
OK though still remains to move `FE_DNS_KEY` to `BOOLEAN_KEYS_NO`, to match the config setting

File: mktxp/cli/config/config.py (hunk 1 above)
Comment (imposibrus) [reply mktxp/cli/config/config.py]: oops. done)

**Pull request merged**